haskell-ide-engine.cabal

```diff
@@ -62,12 +62,13 @@ library
                      , directory
                      , either
                      , fast-logger
-                     , ghc >= 7.10.2 && < 7.11
-                     , ghc-mod >= 5.4
+                     , ghc >= 7.10.2 && < 8.3
```

Comment:
True. I just put something in at the time, without thinking through the details, I guess.